internal/release/scripts/prep-release.sh

```diff
@@ -64,8 +64,8 @@ update_and_validate_version
 # Update and validate the build number
 update_and_validate_build 
 
-if [[ "$current_build_number" -ge "$build" ]]; then
-    echo "Build version hasn't changed or is lower than current build version. Stopping." >&2
+if (( 10#$current_build_number >= 10#$build )); then
+    echo "Build version hasn't changed or is less than current build version. Stopping." >&2
```

Comment:
Nit: we could move this if statement as a whole inside the `update_and_validate_build` function since it also pertains to validating the build. It then makes sense to cluster all the related logic into one single function, especially since that function already exists.

Reply:
Also, by leaving this statement outside of the function block we get the outcome where the build version was detected to be outdated and the script exited with an error message, but the build file was still overwritten.

Reply:
Good catch.. Done!